src/types/onyx/OriginalMessage.ts

```diff
@@ -593,7 +593,7 @@ type OriginalMessageMergedWithCashTransaction = {
     actionName: typeof CONST.REPORT.ACTIONS.TYPE.MERGED_WITH_CASH_TRANSACTION;
 
     /** Content of the original message */
-    originalMessage: Record<string, never>; // No data is sent with this action
+    originalMessage: undefined | null; // No data is sent with this action
```

Comment:
Can we use only undefined here?
```suggestion
    originalMessage: undefined; // No data is sent with this action
```